src/ensembl/src/header/launchbar/Launchbar.scss

```diff
@@ -77,14 +77,26 @@
   }
 }
 
-.about {
+.aboutEnsembl {
   display: flex;
   align-items: center;
-  margin-left: 20px;
+  padding-right: 46px;
+  font-size: 13px;
```

Comment:
This is the default value of our body font; no need to specifically add it here.